lib/Extractor/KLEEBuilder.cpp

```diff
@@ -82,7 +82,7 @@ class KLEEBuilder : public ExprBuilder {
       Printer.setArrayValuesToGet(Arr);
     }
     Printer.generateOutput();
-  
+
```

Comment:
if you want to make whitespace changes, these should go into a separate pull req

Reply:
i have kept all the whitespace changes as a separate commit. When PR is approved, I will squash everything but that commit. So when it gets merged, it will still be separate. For the review purposes, can we just review this specific commit only (clicking the above whitespace commit, eg:). Can create a new PR too if it's hard to review.

Reply:
it isn't hard to review, it's wrong to put whitespace changes into a feature commit-- please do a separate PR

Reply:
ok